keyboards/maartenwut/wasdat/keymaps/via/rules.mk

```
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
```

So that there is a way to trigger via_eeprom_reset. Could be done at the board level.

```suggestion
LTO_ENABLE = yes
BOOTMAGIC_ENABLE = lite
```

---

@zvecr `VIA_ENABLE = yes` will set `BOOTMAGIC_ENABLE := lite`

---

That might be the intention, except it does not work like that
```console
zvecr@zvecr-XPS-L322X:~/Desktop/qmk_firmware$ make lets_split:default VIA_ENABLE=true | grep bootmagic
zvecr@zvecr-XPS-L322X:~/Desktop/qmk_firmware$ make lets_split:default VIA_ENABLE=true BOOTMAGIC_ENABLE=lite | grep bootmagic
Compiling: tmk_core/common/bootmagic_lite.c                                                         [OK]
```

---

It'd be great to get to the bottom of this, but it's out of scope for this PR.

---

@zvecr try again with `VIA_ENABLE=yes`

---

sorry, bad copy paste from my terminal history,
```console
zvecr@zvecr-XPS-L322X:~/Desktop/qmk_firmware$ make lets_split:default VIA_ENABLE=yes BOOTMAGIC_ENABLE=lite | grep bootmagic
Compiling: tmk_core/common/bootmagic_lite.c                                                         [OK]
zvecr@zvecr-XPS-L322X:~/Desktop/qmk_firmware$ make lets_split:default VIA_ENABLE=yes | grep bootmagic
zvecr@zvecr-XPS-L322X:~/Desktop/qmk_firmware$
```

---

I'm testing on `wilba_tech/wt60_d:default` with `BOOTMAGIC_ENABLE = no ` at keyboard level and it seems to be working.

```
zvecr@zvecr-XPS-L322X:~/Desktop/qmk_firmware$ make lets_split:default VIA_ENABLE=yes BOOTMAGIC_ENABLE=lite | grep bootmagic
Compiling: tmk_core/common/bootmagic_lite.c                                                         [OK]
zvecr@zvecr-XPS-L322X:~/Desktop/qmk_firmware$ make lets_split:default VIA_ENABLE=yes | grep bootmagic
zvecr@zvecr-XPS-L322X:~/Desktop/qmk_firmware$
```

FWIW ^^^ if it was done like that, the 2nd build won't need to rebuild `bootmagic_lite.c`, would need a `make clean` in between for a valid test